### ISSUE TITLE
Sidebar Visibility Toggle

### ISSUE THREAD

**Issue by dhochbaum-dcp** [NYCPlanning/labs-zola#1193]
Add a button to the side of the layers menu that allows the user to toggle it between displayed and hidden.

<img width="372" alt="image" src="https://github.com/NYCPlanning/labs-zola/assets/61206501/53434a31-806a-4ea8-ad72-d593c812459b">
->
<img width="372" alt="image" src="https://github.com/NYCPlanning/labs-zola/assets/61206501/bd6d5bdf-591d-45a5-b62b-104340b6855a">

**Comment by jpiacentinidcp** [comment]
Notes from email: Love this and I know many users will be happy about it. I have one design comment and one functionality comment.
 
Design wise, can we find a slightly more elegant UI component for the arrow tab? Ideally it would be slightly smaller and match the color and line weights of the current sidebar. The look could be similar to the X that appears at the top of the Tax Lot or Zoning District window on the right.
 
Functionality wise, we will need to look at the underlying mechanics, as it currently appears to ‘shift’ the entire screen off to the side, including the map object. This leaves a blank white bar on the right side of the browser frame. Ideally this functionality would not move the map at all, just shift the sidebar off screen and reveal more map space underneath.

**Comment by horatiorosa** [comment]
On hold to work on PFF feedback

**Comment by TylerMatteo** [comment]
@bmarchena Can you summarize where you're stuck on this? @dhochbaum-dcp once you have that, please take another look at this.
